todoist/automations/init_env.py

```diff
@@ -17,15 +17,15 @@ def main(config: DictConfig) -> None:
     automations: list[Automation] = hydra.utils.instantiate(config.automations)
     logger.info("Loaded automations: {}", list(map(str, automations)))
     
-    # if Activity among automations, we need to run the longest one.
+    # Run activity automation if present (now using adaptive approach)
     activity_automations: list[Activity] = list(filter(lambda x: isinstance(x, Activity), automations))
```

Comment:
add here some warning or maybe use next and ensure that there are no more activities like that?